crates/ruff/src/rules/pyflakes/snapshots/ruff__rules__pyflakes__tests__init.snap

```diff
@@ -1,4 +1,19 @@
 ---
 source: crates/ruff/src/rules/pyflakes/mod.rs
 ---
+__init__.py:1:8: F401 [*] `os` imported but unused; consider adding to `__all__` or using a redundant alias
+  |
+1 | import os
+  |        ^^ F401
+2 | 
+3 | print(__path__)
+  |
+  = help: Remove unused import: `os`
```

Comment:
It seems like we should suggest `import os as os` instead

Reply:
We do include this in the message ("or using a redundant alias"), unsure what the right suggested fix is...

Reply:
We don't mention removing it in the new message though — perhaps if we say "remove it or consider adding..."

Reply:
Ok, so maybe the suggested fix should be

```diff
- import os
+ import os as os
```
?

Reply:
(Sorry to come back after so long) It's tough because we really want to offer two different fixes here. I guess we should leave the fix as-is, but refine the diagnostic message.